docs/boilerplate/app/theme/Theming.md

```diff
@@ -31,3 +31,151 @@ Timings are defined in `app/theme/timing.ts`. They can be used for consistent an
 Spacing is a first class citizen in Ignite. We use a spacing scale to define the spacing between elements in the app. This allows us to have a consistent spacing scale across the app, and also allows us to change the spacing easily. It is recommended to use the spacing scale for all spacing in the app if possible.
 
 [Spacing](./spacing.ts.md)
+
+# Multiple Themes (a.k.a "Dark Mode")
+
+The Ignite boilerplate ships with color palette definitions and support for multiple themes! By default we define two themes, but you can easily add more using our generic theming system.
+
+:::tip
+
+Head on over to the [Ignite Cookbook](https://ignitecookbook.com/) to find recipes for how to integrate Ignite's theming system with other popular styling and component libraries!
+
+:::
+
+In Ignite `<10` we relied on pure style objects typed as `ViewStyle`, `TextStyle`, `ImageStyle`, etc. This didn't quite fit circumstances where you wanted to dynamically change the overall app's theme during runtime. You could always reload the javascript bundle to see changes, but that was a less than ideal solution.
+
+Ignite now has the concept of `ThemedStyle`s. You can stick with the traditional pure style objects but if you want to take advantage of themed global colors, spacing, typography, and timings, you've got to add a little more code...
+
+Here's an example of how to use the new `useAppTheme()` hook to style your components and dynamically use the correct theme:
+
+```tsx
+import { type ViewStyle, View } from 'react-native'
+import {
+  type ThemedStyle,
+  useAppTheme,
+  ThemeProvider,
+  useThemeProvider
+} from 'app/theme'
+
+const $container: ThemedStyle<ViewStyle> = (theme) => ({
+  flex: 1,
+  backgroundColor: theme.colors.background,
+  justifyContent: "center",
+  alignItems: "center",
+})
+const $normalStyle: ViewStyle = {
+  width: 100,
+  height: 100,
+}
+
+// Then use in a component like so:
+const Component = () => {
+  const { themed } = useAppTheme()
+  return (
+    <View style={themed($container)}>
+      <View style={$normalStyle}>
+    </View>
+  )
+}
+
+const App = () => {
+  const { themeScheme, navigationTheme, setThemeContextOverride, ThemeProvider } =
+    useThemeProvider()
+  return (
+    <ThemeProvider value={{ themeScheme, setThemeContextOverride }}>
+      <Component />
+    </ThemeProvider>
+  )
+}
+```
+
+For more information on the `useAppTheme()`, [check out its documentation](../utils/useAppTheme.tsx.md).
```

Comment:
```suggestion
For more information on the `useAppTheme()` hook, [check out its documentation](../utils/useAppTheme.tsx.md).
```

I also wouldn't mind just linking the docs in the mention of the hook above the example code.